index.html

```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Tykhon</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/SpaceCore912" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/SpaceCore912" target="_blank">Your handle</a>
```

Comment:
Hi, you should also change your handle in anchor tags like this, so that your username is correct in the card.
You can update it by committing and pushing again. Remember to update both the Twitter and github handle

```suggestion
            <a href="https://github.com/SpaceCore912" target="_blank">SpaceCore912</a>
```